block/manager.go

```diff
@@ -205,9 +205,16 @@ func (m *Manager) SyncLoop(ctx context.Context) {
 				atomic.StoreUint64(&m.syncTarget, newHeight)
 				m.retrieveCond.Signal()
 			}
+			m.CommitInCh <- &header.Commit
 		case commit := <-m.CommitInCh:
 			// TODO(tzdybal): check if it's from right aggregator
-			m.lastCommit = commit
+			m.lastCommit.Store(commit)
+			err := m.trySyncNextBlock(ctx, 0)
```

Comment:
Why zero though? the `syncCache` for it will be empty, right?

Reply:
Because we don't know DA height here, as commit was gossiped via P2P.